inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/reduce_ops.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "single_layer_tests/reduce_ops.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+        InferenceEngine::Precision::FP32,
+};
+
+const std::vector<std::vector<size_t>> inputShapes = {
+        std::vector<size_t>{10, 20, 30, 40},
+};
+
+const std::vector<std::vector<int>> axes = {
+        {0},
+        {0, 3},
+        {1, -1},
+};
+const std::vector<ngraph::helpers::ReductionType> reductionTypesArithmetical = {
+        ngraph::helpers::ReductionType::Mean,
```

Comment:
We can combine logical and arithmetic op type into single `vector` because op types are combined in `ngraph helpers`

Reply:
Agree, it was made to skip ReduceLogical tests which can't be instantiated with keepDims parameter, but we already skipping all ReduceLogical tests